benchmark/src/main/java/com/datatorrent/benchmark/window/AbstractWindowedOperatorBenchmarkApp.java

```diff
@@ -121,6 +122,10 @@ protected O createWindowedOperator(Configuration conf)
     }
   }
 
+  protected void setUpdatedDataStorage(O windowedOperator, Configuration conf, SpillableComplexComponentImpl sccImpl)
+  {
```

Comment:
why do nothing here? don't we want to test out the code that utilizes updated data storage?